include/NovelRT/Windowing/WindowMode.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_WINDOWMODE_H
+#define NOVELRT_WINDOWMODE_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Windowing
+{
+    enum class WindowMode : uint32_t
+    {
+        Windowed,
+        Borderless,
+        Fullscreen
```

Comment:
Shouldn't this be spelt FullScreen? :o

Reply:
Most other usages I've seen have been in the form of a compound word. Can definitely change it if you like but I'm pretty sure this is what most people would expect.

Reply:
The compound word would be Full-screen, which we can't do here, and even then that's debated. Any other spelling is basically wrong.

`Full Screen` i.e. `FullScreen` is both more correct and easier to read.